linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/HelpPageHandler.scala

```diff
@@ -0,0 +1,19 @@
+package io.buoyant.linkerd.admin
+
+import com.twitter.finagle.Service
+import com.twitter.finagle.http.{Request, Response, Status}
+import com.twitter.util.Future
+
+private[admin] class HelpPageHandler extends Service[Request, Response] {
+
+  override def apply(req: Request): Future[Response] = AdminHandler.mkResponse(html)
+
+  def html = {
+    AdminHandler.html(
+      content = s"""
+        <div class="container main help-page"></div>
```

Comment:
If we use js / load this from a file, then we can make changes to the html and not have to rebuild to pick up changes!